libs/common/src/sql-db-entities/fee-record/fee-record.entity.ts

```diff
@@ -103,6 +103,7 @@ export class FeeRecordEntity extends AuditableBaseEntity {
    */
   @ManyToMany(() => PaymentEntity, (payment) => payment.feeRecords, {
     cascade: ['insert'],
+    onDelete: 'CASCADE',
```

Comment:
Just to check have we tested that deleting a payment deletes the connection to the fee record but no the fee record itself and vice versa?

Reply:
I have confirmed that, at least in terms of running SQL commands, the cascade delete only deletes the join table reference